tests/test_operation.py

```diff
@@ -196,6 +195,29 @@ def test_operation_inside_context_do_queue_false(self, qnode):
         assert qnode.ops[2].name == "PauliX"
         assert qnode.ops[3].name == "PauliZ"
 
+    @pytest.fixture(scope="function")
+    def qnode_for_inverse(self, mock_device):
```

Comment:
I think if it's only used once it need not be a fixture

Reply:
Here the previous ``qnode`` example was followed from the previous test case. What would be the modification that you would suggest to make?